kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeStagingDataTab.js

```diff
@@ -52,11 +52,21 @@ define([
         },
 
         activate: function () {
-            this.stagingAreaViewer.activate();
+            if (this.stagingAreaViewer) {
+                this.stagingAreaViewer.activate();
+            } else {
+                console.error('Call "render" before "activate"');
+            }
         },
 
         deactivate: function () {
-            this.stagingAreaViewer.deactivate();
+            if (this.stagingAreaViewer) {
+                this.stagingAreaViewer.deactivate();
+            }
+            if (this.updateTimeout) {
```

Comment:
The staging data tab could probably use some restructuring. But this was being problematic.